hrm-domain/hrm-core/contact/contactRoutesV0.ts

```diff
@@ -127,36 +127,40 @@ contactsRouter.post('/search', publicEndpoint, async (req, res) => {
   res.json(searchResults);
 });
 
-// Endpoint used for generalized search with ElasticSearch
+// Endpoint used for generalized search powered by ElasticSearch
 contactsRouter.post(
   '/searchV2',
```

Comment:
Not sure if I'm super keen on 'V2' especially if we have a version further up the endpoint path that we never use - a more descriptive name would be preferable - /fullSearch /generalisedSearch even /esSearch. Not a hill I'll die on though :-)